packages/components/src/navigation/index.js

```diff
@@ -73,13 +73,14 @@ const Navigation = ( { activeItemId, children, data, rootTitle } ) => {
 		}
 
 		return (
-			<Button
+			<BackButtonUi
```

Comment:
Other names have a capitalized `I` as in `BackButtonUI`.

Reply:
Good catch.